src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1585,7 +1585,6 @@ f1_edcfu_epda,112,1994,Line 112,,,,
 f1_edcfu_epda,113,1994,Line 113,,,,
 f1_edcfu_epda,114,1994,Line 114,,,,
 f1_edcfu_epda,115,1994,Line 115,,,,
-f1_elc_op_mnt_expn,171,2006,TOTAL Customer Service and Information Expenses (Total 167 thru 170),calculated_value,customer_service_and_information_expenses,,
```

Comment:
I moved this record to where it should be in the ordered table. Not sure why it showed up here.